flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinatorTest.java

```diff
@@ -1367,13 +1503,18 @@ public void testMinTimeBetweenCheckpointsInterval() throws Exception {
 
 		final long delay = 50;
 
+		CheckpointCoordinatorConfiguration chkConfig = new CheckpointCoordinatorConfiguration(
+			12,           // periodic interval is 12 ms
```

Comment:
Why are constants in the test changed? Is this required?

Reply:
Yes, it is required. Just like the prior comment, I have changed the condition of sanity checks in `CheckpointCoordinatorConfiguration.java`. We should have the same criterion, the correct condition is interval must be larger than or equal to 10.